packages/contracts/src/networks.ts

```diff
@@ -1178,6 +1178,6 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     legacyTx: false,
     actionGasLimitBuffer: false,
     eip2028: true,
-    hardcodedMerkleLeafGas: (14_000_000).toString(),
+    hardcodedMerkleLeafGas: (11200000).toString(),
```

Comment:
In what situation was the previous value too high? Weren't you able to deploy `MyLargeContract` with it?

Reply:
This bug occurred because of the following happened

First, in my PR to fix the original Pangolin issue, I set the hardcoded limit on Pangolin to 14 million. I also set the field `useHigherMaxGasLimit` to true on Pangoline. On Pangolin the block gas limit is 20 million. As a result, the maximum batch size on Pangolin was 20 million * 0.8 = 16 million. That is significantly higher than the hardcoded limit of 14 million, so everything thing works. 

Then your rootstock gas limit PR removed the `useHigherMaxGasLimit` field and replaced it with a function that dynamically sets the max batch size based on the block gas limit on the target network. If the target network block gas limit is greater than 13.5 million than the max batch size was set to 50% of the block gas limit. So on Pangolin the max batch size was 20 million * 0.5 = 10 million which is less than the hardcoded value of 14 million. As a result, it became impossible to propose any deployments on Pangolin b/c the hardcoded gas limit was always higher than the maximum batch size. 

It's worth noting that since the simulation tests are super flaky and I'm used to them failing, I just went ahead and did the release for the rootstock PR even though they were not passing (they failed due to another issue, not specifically this). So I also should have been more patient and retried the simulation tests before doing the release.